### ISSUE TITLE
Performance of Resnet50

### ISSUE THREAD

**Issue by lailvlong** [kaixin96/PANet#37]
Hello, thanks for open source your nice work! Did you explore other encoder backbones? For example, resnet50. I used resnet50 as encoder but got worse mIOU, specifically 40.7%. How do you think of this?

**Comment by kaixin96** [comment]
Hi @lailvlong , I tried resnet50 before, and as far as I can remember, the performance is slightly better (~1%). According to some recent papers I read, the performance gap between vgg16 and resnet50/101 should be larger. Maybe it requires more careful tuning.

Regarding your results, changing backbone to resnet50 should not cause much performance degradation. Did you use an ImageNet-pretrained resnet50? Or you may try tuning some hyper-parameters (e.g. learning rate, training epochs)

Thank you.

**Comment by MohamedGAmin** [comment]
@kaixin96 May I know how did you change the backbone to another model than the vgg as I would like to try something different.
